src/modules/tests/tests.selector.ts

```diff
@@ -91,12 +91,12 @@ export const getActivityCodeBySlotId = (testsModel: TestsModel, id: number): Act
 
 export const getIncompleteTestsSlotIds = (tests: TestsModel): string[] => {
   return Object.keys(tests.testStatus).filter(slotId =>
-    isTestBeforeToday(tests.startedTests[slotId])
+    isTestBeforeToday(tests.startedTests[slotId] as TestResultCommonSchema)
```

Comment:
Is the `as TestResultCommonSchema` needed?

Reply:
yeah, otherwise it's a Union and `isTestBeforeToday` requires common